### DISCUSSION TITLE
Report malicious actors?

### DISCUSSION THREAD

**Discussion by Chaython** [community/community#19405] [Code Security]
Someone on discord was trying to trick people into running this malicious program.
https://github.com/HaderBeso/Shy-Bruter-forcer
How can I report the repo/user.

**Comment by 3ICE** [comment]
On their user profile, https://github.com/HaderBeso, click **Block or Report**, then _Report abuse_: 

You are submitting a report regarding HaderBeso.
Please select from one the following categories:
I want to report harmful code, such as malware, phishing, or cryptocurrency abuse.
I want to report malware.
Malware is any code designed to cause damage to a computer, server, client, or computer network. This can include viruses, worms, Trojans, ransomware, spyware, adware, and bots. To learn more about what types of Malware are allowed on Github, please visit our [Community Guidelines](https://docs.github.com/github/site-policy/github-community-guidelines#misinformation-and-disinformation).

In the [form below](https://support.github.com/contact/report-abuse?category=report-abuse&report=HaderBeso&report_id=107855804&report_type=user#contact_comments).

Subject
Report Abuse: HaderBeso (user)
What would you like to report?
Someone on discord was trying to trick people into running this malicious program.
https://github.com/HaderBeso/Shy-Bruter-forcer
No file chosen
Attach files by dragging & dropping, or selecting them.

**Comment by overtube** [comment]
This repo: `https://github.com/openhardwaremonitor/openhardwaremonitor` when installing on system, using system package manager (that have chroot), after ending installation, post install executing hook that that placing on `/usr/share` directory, **144** `ssh` `root` **certificates**. Offcource code safe and program worked well, but post install activity make malicious, thing because not ask user for trust or not and just wilfully install root certificates.
